infra/scripts/examples.toml

```diff
@@ -0,0 +1,16 @@
+[tasks.simplecoin-example]
+# Using --release in the hope that it can reuse artifacts compiled earlier for the docker build.
+script = """
+cd ${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}
+cargo run -p fendermint_rpc --release --example simplecoin -- \
+  --secret-key fendermint/testing/smoke-test/test-data/fendermint/keys/alice.sk \
+  ${VERBOSITY}
+"""
+
+[tasks.ethapi-example]
+script = """
+cd ${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}
+cargo run -p fendermint_eth_api --release --example ethers -- \
+  --secret-key-from fendermint/testing/smoke-test/test-data/fendermint/keys/emily.sk \
+  --secret-key-to   fendermint/testing/smoke-test/test-data/fendermint/keys/eric.sk
```

Comment:
Does this setup crate these keys at all?

Reply:
No. I was waiting for Alfonso's on example. Removed not to add more questions.